Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -6768,7 +6768,11 @@ void HTMLMediaElement::enterFullscreen(VideoFullscreenMode mode)
     m_changingVideoFullscreenMode = true;
 
 #if ENABLE(FULLSCREEN_API) && ENABLE(VIDEO_USES_ELEMENT_FULLSCREEN)
-    if (document().settings().fullScreenEnabled() && mode == VideoFullscreenModeStandard) {
+    bool videoUsesElementFullscreen = true;
+#if PLATFORM(IOS_FAMILY)
```

Comment:
I'm guessing this is `PLATFORM(IOS_FAMILY)` in case we set `ENABLE_VIDEO_USES_ELEMENT_FULLSCREEN` to `1` for iOS (as it is currently 0)?

Reply:
I used `PLATFORM(IOS_FAMILY)` simply because `videoFullscreenRequiresElementFullscreen` is behind that.